lib/generators.nix

```diff
@@ -175,4 +175,52 @@ rec {
                          else "<λ:{${showFnas}}>"
     else abort "toPretty: should never happen (v = ${v})";
 
+  # PLIST handling
+
+  toPLIST = x: ''
```

Comment:
`toX` functions should all take an attrset before the input value, which is an attribute set of options (possibly added in the future). So

```
toPlist = {}: x: ''
```

I think `plist` is normally written with small letters (going by the [wiki article](https://en.wikipedia.org/wiki/Property_list)), so I’d camel case.

The next step is thinking about all parts of the generator and moving everything that is not fixed by the generated format into an option. For example `mkINI` doesn’t specify how to format names in section headers (mainly what escaping is used, `mkSectionName`) and does not fix how key-value-pairs look, only the fact that they exist (the formatting of them is relayed to `mkKeyValueDefault`, another generator).